jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarFiles.java

```diff
@@ -55,7 +55,7 @@ public static JibContainerBuilder toJibContainerBuilder(
         jarOptions.getFrom().isPresent()
             ? ContainerBuilders.create(
                 jarOptions.getFrom().get(), Collections.emptySet(), commonCliOptions, logger)
-            : Jib.from("gcr.io/distroless/java");
+            : Jib.from("gcr.io/distroless/java:11");
```

Comment:
Just curious: is this the debian10 one?

Reply:
No. Now you said it, I think we should use `gcr.io/distroless/java-debian10:11` at least for Java 11.

Currently, the build plugins use `gcr.io/distroless/java:11` in the hope that the default entrypoint `gcr.io/distroless/java:11` would eventually roll over to Debian 10. However, I don't think it'll happen, so for Jib CLI, we can start with `gcr.io/distroless/java-debian10:11`.

Reply:
Thanks for bringing this up. I'll change the default to `gcr.io/distroless/java-debian10:11 `